### PR TITLE
*: add docs of backup and restore using Dumpling and Lightning

### PR DESCRIPTION
**Pull request by shichun-0415**
… and lightning

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: backup-and-restore-using-dumpling-and-lightning.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+title: 使用 Dumpling/TiDB Lightning 备份与恢复
```
Comment (lilin90) [inline backup-and-restore-using-dumpling-and-lightning.md:2]:
```suggestion
title: 使用 Dumpling 和 TiDB Lightning 备份与恢复
```

File: backup-and-restore-using-dumpling-and-lightning.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+summary: 了解如何使用 Dumpling 和 TiDB Lightning 备份与恢复集群数据。
+---
+
+# 使用 Dumpling/TiDB Lightning 备份与恢复
```
Comment (lilin90) [inline backup-and-restore-using-dumpling-and-lightning.md:6]:
```suggestion
# 使用 Dumpling 和 TiDB Lightning 备份与恢复
```

File: backup-and-restore-using-dumpling-and-lightning.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+本文档介绍如何使用 Dumpling 和 TiDB Lightning 进行全量备份与恢复。
+
+在备份恢复过程中，我们会使用 [Dumpling](/dumpling-overview.md) 从 TiDB 导出数据进行备份，然后使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 导入数据至 TiDB 实现恢复。
+
+## 前提条件
+
+- [安装 Dumpling 和 TiDB Lightning。](/migration-tools.md)
```
Comment (lilin90) [inline backup-and-restore-using-dumpling-and-lightning.md:14]: Please provide the more friendly anchor link.

File: backup-and-restore-using-dumpling-and-lightning.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    port = ${port}                # 例如：4000
+    user = "${user_name}"         # 例如："root"
+    password = "${password}"      # 例如："rootroot"
+    status-port = ${status-port}  # 导入过程 Lightning 需要在从 TiDB 的“状态端口”获取表结构信息，例如：10080
+    pd-addr = "${ip}:${port}"     # 集群 PD 的地址，Lightning 通过 PD 获取部分信息，例如 172.16.31.3:2379。当 backend = "local" 时 status-port 和 pd-addr 必须正确填写，否则导入将出现异常。
+    ```
+
+    关于更多 TiDB Lightning 的配置，请参考 [TiDB Lightning 配置参数](/tidb-lightning/tidb-lightning-configuration.md)。
+
+2. 运行 `tidb-lightning`。如果直接在命令行中启动程序，可能会因为 `SIGHUP` 信号而退出，建议配合`nohup`或`screen`等工具，如：
```
Comment (lilin90) [inline backup-and-restore-using-dumpling-and-lightning.md:115]:
```suggestion
2. 运行 `tidb-lightning`。如果直接在命令行中启动程序，可能会因为 `SIGHUP` 信号而退出，建议配合 `nohup` 或 `screen` 等工具，如：
```

File: TOC.md [hunk 5]
```diff
@@ -122,9 +122,12 @@
     - [使用 TiUP（推荐）](/scale-tidb-using-tiup.md)
     - [使用 TiDB Operator](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/scale-a-tidb-cluster)
   - 备份与恢复
-    - [使用 BR 备份集群](/br/br-usage-backup.md)
-    - [使用 BR 恢复集群](/br/br-usage-restore.md)
-    - [BR 备份恢复场景示例](/br/backup-and-restore-use-cases.md)
+    - [备份与恢复综述](/br-overview.md)
```
Comment (lilin90) [inline TOC.md:125]:
```suggestion
    - [备份与恢复概述](/br-overview.md)
```

File: br-overview.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+---
+title: 备份与恢复工具
```
Comment (lilin90) [inline br-overview.md:2]:
```suggestion
title: 备份与恢复概述
```

File: br-overview.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+summary: 了解备份与恢复的场景和工具选择。
+---
+
+# 备份与恢复工具
```
Comment (lilin90) [inline br-overview.md:6]:
```suggestion
# 备份与恢复概述
```

File: br-overview.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
```
Comment (lilin90) [inline br-overview.md:7]: Please add a brief intro to this document here.

File: backup-and-restore-using-dumpling-and-lightning.md (hunk 3 above)
Comment (shichun-0415) [reply backup-and-restore-using-dumpling-and-lightning.md]: Made modification in https://github.com/pingcap/docs-cn/pull/10286/files#diff-5f4635ca14610f49f1a1c3d5c990ea927ae0bf6e26c6dd6b4f545021c5a0b7c6R14-R15. PTAL.

File: backup-and-restore-using-dumpling-and-lightning.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+
+**操作系统**：本文档示例使用的是若干新的、纯净版 CentOS 7 实例，你可以在本地虚拟化一台主机，或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程中，默认会占满 CPU，建议单独部署在一台主机上。如果条件不允许，你可以将 TiDB Lightning 和其他组件（比如 `tikv-server`）部署在同一台机器上，然后设置 `region-concurrency` 配置项的值为逻辑 CPU 数的 75%，以限制 TiDB Lightning 对 CPU 资源的使用。
+
+**内存和 CPU**：因为 TiDB Lightning 对计算机资源消耗较高，建议分配 64 GB 以上的内存以及 32 核以上的 CPU，而且确保 CPU 核数和内存（GB）比为 1:2 以上，以获取最佳性能。
+
+**磁盘空间**：
+
+- Dumpling 需要能够储存整个数据源的存储空间，即可以容纳要导出的所有上游表的空间。计算方式参考[下游数据库所需空间](/tidb-lightning/tidb-lightning-requirements.md#下游数据库所需空间)。
+- TiDB Lightning 导入期间，需要临时空间来存储排序键值对，磁盘空间需要至少能存储数据源的最大单表。
+- 若全量数据量较大，可适当加长上游 binlog 保存时间，以避免增量同步时缺必要 binlog 导致重做。
```
Comment (IANTHEREAL) [inline backup-and-restore-using-dumpling-and-lightning.md:29]: 可以去掉，binlog 与备份恢复无关

File: backup-and-restore-using-dumpling-and-lightning.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+- [获取 Dumpling 所需上游数据库权限](/dumpling-overview.md#从-tidbmysql-导出数据)
+- [获取 TiDB Lightning 所需下游数据库权限](/tidb-lightning/tidb-lightning-requirements.md#下游数据库权限要求)
+
+## 资源要求
+
+**操作系统**：本文档示例使用的是若干新的、纯净版 CentOS 7 实例，你可以在本地虚拟化一台主机，或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程中，默认会占满 CPU，建议单独部署在一台主机上。如果条件不允许，你可以将 TiDB Lightning 和其他组件（比如 `tikv-server`）部署在同一台机器上，然后设置 `region-concurrency` 配置项的值为逻辑 CPU 数的 75%，以限制 TiDB Lightning 对 CPU 资源的使用。
+
+**内存和 CPU**：因为 TiDB Lightning 对计算机资源消耗较高，建议分配 64 GB 以上的内存以及 32 核以上的 CPU，而且确保 CPU 核数和内存（GB）比为 1:2 以上，以获取最佳性能。
+
+**磁盘空间**：
```
Comment (IANTHEREAL) [inline backup-and-restore-using-dumpling-and-lightning.md:25]: 示例用的是 S3，S3/GCS 等 cloud storage 服务的存储空间是无限的。 磁盘空间这里要求可以改一下，建议使用 S3/GCS/NFS 等存储系统，如果保存单次备份到本地磁盘，则对磁盘的要求...

File: backup-and-restore-using-dumpling-and-lightning.md [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+
+# 使用 Dumpling 和 TiDB Lightning 备份与恢复
+
+本文档介绍如何使用 Dumpling 和 TiDB Lightning 进行全量备份与恢复。
+
+在备份恢复过程中，[Dumpling](/dumpling-overview.md) 用于从 TiDB 导出数据进行备份，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 用于导入数据至 TiDB 实现恢复。
+
+## 前提条件
+
+- [下载和安装 Dumpling](/download-ecosystem-tools.md)
```
Comment (IANTHEREAL) [inline backup-and-restore-using-dumpling-and-lightning.md:14]: 推荐使用 TiUP 吧，不要下载了

File: backup-and-restore-using-dumpling-and-lightning.md [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+    [tikv-importer]
+    # "local"：默认使用该模式，适用于 TB 级以上大数据量，但导入期间下游 TiDB 无法对外提供服务。
+    # "tidb"：TB 级以下数据量也可以采用`tidb`后端模式，下游 TiDB 可正常提供服务。 关于后端模式更多信息请参阅：https://docs.pingcap.com/tidb/stable/tidb-lightning-backends
+    backend = "local"
+    # 设置排序的键值对的临时存放地址，目标路径必须是一个空目录，目录空间须大于待导入数据集的大小。建议设为与 `data-source-dir` 不同的磁盘目录并使用闪存介质，独占 IO 会获得更好的导入性能
+    sorted-kv-dir = "${sorted-kv-dir}"
+
+    [mydumper]
+    # 源数据目录，即上一章节中 Dumpling 保存数据的路径。
+    data-source-dir = "${data-path}" # 本地或 S3 路径，例如：'s3://my-bucket/sql-backup?region=us-west-2'
```
Comment (IANTHEREAL) [inline backup-and-restore-using-dumpling-and-lightning.md:102]: 建议直接替换成上面备份数据的目录

File: backup-and-restore-using-dumpling-and-lightning.md [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+
+1. 运行以下命令，从 TiDB 导出全量数据：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MiB -B my_db1 -f 'my_db1.table[12]' -o 's3://my-bucket/sql-backup?region=us-west-2'
+    ```
+
+    Dumpling 默认导出数据格式为 SQL 文件，你也可以通过设置 `--filetype` 指定导出文件的类型。
```
Comment (IANTHEREAL) [inline backup-and-restore-using-dumpling-and-lightning.md:60]: lightning 每介绍参数，建议 dumpling 也不介绍了，可以连接到 dumpling 命令行介绍文档

File: backup-and-restore-using-dumpling-and-lightning.md (hunk 12 above)
Comment (shichun-0415) [reply backup-and-restore-using-dumpling-and-lightning.md:102]: @IANTHEREAL 似乎就是上面的备份目录？https://github.com/pingcap/docs-cn/pull/10286/files#diff-5f4635ca14610f49f1a1c3d5c990ea927ae0bf6e26c6dd6b4f545021c5a0b7c6R57
image

Comment (IANTHEREAL) [reply backup-and-restore-using-dumpling-and-lightning.md:102]: 我建议不要写在注释里

File: backup-and-restore-using-dumpling-and-lightning.md (hunk 11 above)
Comment (shichun-0415) [reply backup-and-restore-using-dumpling-and-lightning.md]: @IANTHEREAL Fixed in https://github.com/pingcap/docs-cn/pull/10286/files#diff-5f4635ca14610f49f1a1c3d5c990ea927ae0bf6e26c6dd6b4f545021c5a0b7c6R14-R22.

File: backup-and-restore-using-dumpling-and-lightning.md (hunk 12 above)
Comment (shichun-0415) [reply backup-and-restore-using-dumpling-and-lightning.md:94]: > 我建议不要写在注释里

提到了正文里：https://github.com/pingcap/docs-cn/pull/10286/files#diff-5f4635ca14610f49f1a1c3d5c990ea927ae0bf6e26c6dd6b4f545021c5a0b7c6R77

File: backup-and-restore-using-dumpling-and-lightning.md [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+
+- 安装和运行 Dumpling：
+
+    ```
+    tiup install dumpling && tiup dumpling
+    ```
+
+- 安装和运行 TiDB Lightning：
+
+    ````
```
Comment (IANTHEREAL) [inline backup-and-restore-using-dumpling-and-lightning.md:22]:
```suggestion
    ```
```

File: backup-and-restore-using-dumpling-and-lightning.md [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+    ```shell
+    export AWS_ACCESS_KEY_ID=${access_key}
+    export AWS_SECRET_ACCESS_KEY=${secret_key}
+    nohup tiup tidb-lightning -config tidb-lightning.toml > nohup.out 2>&1 &
+    ```
+
+3. 导入开始后，可以采用以下任意方式查看进度：
+
+   - 通过 `grep` 日志关键字 `progress` 查看进度，默认 5 分钟更新一次。
+   - 通过监控面板查看进度，请参考 [TiDB Lightning 监控](/tidb-lightning/monitor-tidb-lightning.md)。
```
Comment (IANTHEREAL) [inline backup-and-restore-using-dumpling-and-lightning.md:121]: 我理解按照我么这篇文档部署方式，监控和 web 页面都不能看到吧

File: br/br-usage-backup.md [hunk 16]
```diff
@@ -15,10 +15,12 @@ summary: 了解如何使用 BR 命令行进行数据备份。
...
 
-如果你还不熟悉 Backup & Restore (BR)，建议先阅读以下文档，充分了解 BR 使用限制和方法：
+如果你还不熟悉备份工具，建议先阅读以下文档，充分了解备份工具的使用方法和限制：
 
-- [BR 工具简介](/br/backup-and-restore-overview.md)
-- [BR 命令行介绍](/br/use-br-command-line-tool.md)
+- [备份工具简介](/br/backup-and-restore-overview.md)
+- [备份工具命令行介绍](/br/use-br-command-line-tool.md)
+
+如果需要全量备份少量数据（例如小于 50 GB），且不要求备份速度，可使用 Dumpling 导出数据，实现备份。具体备份操作，参考[使用 Dumpling 备份全量数据](/backup-and-restore-using-dumpling-and-lightning.md#使用-dumpling-备份全量数据)。
```
Comment (IANTHEREAL) [inline br/br-usage-backup.md:23]:
```suggestion
如果需要全量备份少量数据（例如小于 50 GB），且不要求备份速度，也可以选择 Dumpling 导出数据，实现备份。具体备份操作，参考[使用 Dumpling 备份全量数据](/backup-and-restore-using-dumpling-and-lightning.md#使用-dumpling-备份全量数据)。
```

File: br/br-usage-restore.md [hunk 17]
```diff
@@ -16,10 +16,12 @@ summary: 了解如何使用 BR 命令行恢复备份数据。
...
 
-如果你还不熟悉 Backup & Restore (BR)，建议先阅读以下文档，充分了解 BR 使用限制和方法
+如果你还不熟悉恢复工具，建议先阅读以下文档，充分了解恢复工具的使用方法和限制：
 
-- [BR 工具简介](/br/backup-and-restore-overview.md)
-- [BR 命令行介绍](/br/use-br-command-line-tool.md)
+- [恢复工具简介](/br/backup-and-restore-overview.md)
+- [恢复工具命令行介绍](/br/use-br-command-line-tool.md)
+
+如果你需要恢复 Dumpling 导出的数据、CSV 文件或 Amazon Aurora 生成的 Apache Parquet 文件，且数据恢复不基于指定时间，则可以使用 TiDB Lightning 来导入数据，实现恢复。具体恢复操作，请参考 [使用 TiDB Lightning 恢复全量数据](/backup-and-restore-using-dumpling-and-lightning.md#使用-tidb-lightning-恢复全量数据)。
```
Comment (IANTHEREAL) [inline br/br-usage-restore.md:24]:
```suggestion
如果你需要恢复 Dumpling 导出的数据、CSV 文件或 Amazon Aurora 生成的 Apache Parquet 文件，、可以使用 TiDB Lightning 来导入数据，实现恢复。具体恢复操作，请参考 [使用 TiDB Lightning 恢复全量数据](/backup-and-restore-using-dumpling-and-lightning.md#使用-tidb-lightning-恢复全量数据)。
```

**Comment by IANTHEREAL** [comment]
Rest LGTM

File: br/br-usage-restore.md (hunk 17 above)
Comment (IANTHEREAL) [reply br/br-usage-restore.md:24]: 不能基于时间的事情于 lightning 来说是毫无相关性的

File: backup-and-restore-using-dumpling-lightning.md [hunk 18]
```diff
@@ -0,0 +1,126 @@
...
+
+本文档介绍如何使用 Dumpling 和 TiDB Lightning 进行全量备份与恢复。
+
+在备份与恢复场景中，如果需要全量备份少量数据（例如小于 50 GB），且不要求备份速度，你可以使用 [Dumpling](/dumpling-overview.md) 从 TiDB 数据库导出数据进行备份，再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入至 TiDB 数据库实现恢复。更多备份与恢复的相关信息，参见[使用 BR 备份集群](/br/br-usage-backup.md)与[使用 BR 恢复集群](/br/br-usage-restore.md)。
+
+## 前提条件
+
+- 安装和运行 Dumpling：
+
+    ```
```
Comment (lilin90) [inline backup-and-restore-using-dumpling-lightning.md:16]:
```suggestion
    ```shell
```

File: backup-and-restore-using-dumpling-lightning.md [hunk 19]
```diff
@@ -0,0 +1,126 @@
...
+    tiup install dumpling && tiup dumpling
+    ```
+
+- 安装和运行 TiDB Lightning：
+
+    ```
```
Comment (lilin90) [inline backup-and-restore-using-dumpling-lightning.md:22]:
```suggestion
    ```shell
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**